src/features/DisplayControls/Treatments/utils/TreatmentsUtils.js

```diff
@@ -148,7 +148,7 @@ export const getDrugName = (drugOrderObject) => {
   ) {
     return (
```

Comment:
You should add approver notes condition for notes icon, so when no instructions and additional Instructions are given but approver notes are present then it should show notes icon.